docs/manual-EN/manual-introduction.md

```diff
@@ -0,0 +1,75 @@
+# Nebula Graph R201910_beta User Manual
+
+### About this manual
+
+This is the Nebula Graph User Manual. It documents Nebula Graph R201910_beta. It may include documentation of features of Nebula Graph versions that have not yet been released. For information about which versions have been released, see  Nebula Graph Release Notes.
+<!-- Release Notes not added yet. -->
+
+Because this manual serves as a reference, it does not provide general instruction on  graph database concepts. It also does not teach you how to use your operating system or command-line interpreter.
```

Comment:
Not necessary. This is a manual for programmers, they know what is operating system or command-line. 
Or you can say, the doc presumes that you understand general instruction of graph database concepts as well as command-lines.